build.proj

```diff
@@ -116,29 +129,45 @@
     <!--Note, we use "Rebuild" rather "Build", because code analysis happens during the build, and this ensures the build is not skipped-->
     <MSBuild Projects="$(ClientRuntimeSolution)"
              Properties="Configuration=%(LibraryFxTargetList.Identity)-$(Configuration);Platform=Any CPU;RunCodeAnalysis=true;CodeAnalysisTreatWarningsAsErrors=true"
-             Targets="Rebuild" />
+             Targets="Rebuild"
+             Condition=" '$(Scope)' == 'all' or '$(Scope)' == 'ClientRuntime'" />
              
     <MSBuild Projects="$(CodeGeneratorSolution)"
              Properties="Configuration=Net45-$(Configuration);Platform=Any CPU;RunCodeAnalysis=true;CodeAnalysisTreatWarningsAsErrors=true"
-             Targets="Rebuild" />
+             Targets="Rebuild"
+             Condition=" '$(Scope)' == 'all' or '$(Scope)' == 'CodeGenerator'" />
```

Comment:
Why is the scope for building "AutoRest" named "CodeGenerator" instead of the more natural and intuitive "AutoRest"?